src/strategies/DSL/conditions/intervalCondition.ts

```diff
@@ -1,11 +1,26 @@
 import { PrimitiveArgs } from '@brinkninja/types'
 import { IntervalConditionArgs } from '@brink-sdk'
 
-function intervalCondition (def: IntervalConditionArgs): PrimitiveArgs[] {
+const DEFAULT_MAX_INTERVALS = 0
+const DEFAULT_INITIAL_START_BLOCK = 0
 
-  // TODO: implement
-
-  return []
+function intervalCondition ({
+  id,
+  intervalMinBlocks,
```

Comment:
these names should all match the docs

Reply:
ah didn't see the mismatch. do you mean we should change brink-types to match the docs?